src/json/serialization.cr

```diff
@@ -270,60 +270,70 @@ module JSON
         {% properties = {} of Nil => Nil %}
         {% for ivar in @type.instance_vars %}
           {% ann = ivar.annotation(::JSON::Field) %}
-          {% unless ann && (ann[:ignore] || ann[:ignore_serialize]) %}
+          {% unless ann && ann[:ignore] %}
```

Comment:
We could leave the guard for `ignore_serialize` in and just check it's a `true` literal. This makes the extra conditional below unnecessary.
```suggestion
          {% unless ann && (ann[:ignore] || (ann[:ignore_serialize] == true) )%}
```